### DISCUSSION TITLE
Tolerations attribute under job spec

### DISCUSSION THREAD

**Discussion by Archangelix** [spotify/flink-on-k8s-operator#196] [Q&A]
Hi,
I checked the flinkclusters CRD and I see that the tolerations attribute are specified under spec.jobManager and spec.taskManager attribute, but not under spec.job attribute. 

Is there a way to specify this value from within the CR yaml file? Right now the pod to submit the job cannot be allocated in our cluster due to this missing tolerations value.

**Comment by regadas** [comment]
Hi @Archangelix I believe this was already added in #242

**Reply by Archangelix** [reply]
Yep, actually I posted the request here at first. Since there wasn't any response then I posted it in 'Issues' section. Apparently this isn't the right place to put a request. Thanks for the response anyway :)
